### ISSUE TITLE
Brightness and contrast accessibility fails

### ISSUE THREAD

**Issue by dittaeva** [waymarkedtrails/waymarked-trails-site#87]
A few brigthness and contrast accessibility fails and many warnings at [AccessColor](http://www.accesskeys.org/tools/color-contrast.html) for hiking. Probably similar for the other categories. If a new design isn't planned it might be worth looking into.

[This tool](http://snook.ca/technical/colour_contrast/colour.html) might be useful for tweaking, and there are [others](http://www.w3.org/WAI/RC/tools/complete).

**Comment by lonvia** [comment]
If somebody wants to tinker with the colors, I'm open for suggestions but it is unlikely that I will be able to do anything about it.
